docs/best-practices/coding.md

```diff
@@ -1,3 +1,12 @@
+{% set title = "Best Practices: Coding" %}
+<span id="title" class="d-none">{{ title }}</span>
```

Comment:
@kaixin-hc  Should this be here? I thought you are going to get rid of these. Noticed a few similar cases in a few other places.

Reply:
Thanks, I think I missed those because they were in a subfolder. Have removed them now.